modules/op-matching-pipeline-run-using-pipelines-as-code.adoc

```diff
@@ -49,6 +49,61 @@ annotations:
 * Globs with pattern matching such as `"refs/heads/\*"`
 * Tags such as `"refs/tags/1.\*"`
 
+
+[id="path-matching-pipeline-run-using-pipelines-as-code_{context}"]
+== Matching changes in paths to a pipeline run
+
+You can match a pipeline run to changes in a a set of paths. {pac} starts the pipeline run when a pull request includes changes in any of the paths that you list.
+
+The `pass:[*]` wildcard denotes any file in the directory. The `pass:[**]` wildcard denotes any file in the directory or any subdirectories on any level under the directory.
```

Comment:
pass? what does this mean ?

Reply:
pass is an internal asciidoc thing. The * symbol is also used for bold, so just `*` or `**` was causing build errors. If you look at the preview linked at the top of the PR, you will see that the wildcards appear as * and **.